docs/environment.yml

```diff
@@ -0,0 +1,9 @@
+name: black_docs
+channels:
+  - conda-forge
+dependencies:
+- python==3.6
```

Comment:
Would `>=3.6` not work? Then we'll get 3.6.4 or 3.6.5 which is preferable :)

Reply:
Sure that would work